vendors/circom/circomlib/base/fr_element_conversion.h

```diff
@@ -0,0 +1,30 @@
+#ifndef VENDORS_CIRCOM_CIRCOMLIB_BASE_FR_ELEMENT_CONVERSION_H_
+#define VENDORS_CIRCOM_CIRCOMLIB_BASE_FR_ELEMENT_CONVERSION_H_
+
+#include <string.h>
+
+#include "fr.hpp"  // NOLINT(build/include_subdir)
```

Comment:
Where is this header coming from?

Reply:
This file is derived from https://github.com/iden3/circom/blob/master/code_producers/src/c_elements/bn128/fr.hpp, and in our case, this is generated by bazel.